src/python/sdk/examples/trade.py

```diff
@@ -630,7 +731,7 @@ def report_fill_events(fill_events: list[dict]):
 
 
 def report_order_for_last_fill(fill_events: list[dict], open_orders: list[dict]):
-    order_id = fill_events[-1]["data"]["taker_order_id"]
+    order_id = fill_events[-1]["data"]["taker_order_id"]  # type: ignore
```

Comment:
I find this comment confusing

If we are supposed to ignore it, can this line be delete? What is going on here?